src/extension.ts

```diff
@@ -38,11 +39,12 @@ export async function activateInternal(context: ExtensionContext, perfStats: { l
 	await callWithTelemetryAndErrorHandling('azure-account.activate', async (activateContext: IActionContext) => {
 		activateContext.telemetry.properties.isActivationEvent = 'true';
 		activateContext.telemetry.properties.activationTime = String((perfStats.loadEndTime - perfStats.loadStartTime) / 1000);
-		activateContext.telemetry.properties.authLibraryOnStartup = String(getSettingValue(authLibrarySetting));
 
 		ext.experimentationService = await createExperimentationService(context);
 		ext.loginHelper = new AzureAccountLoginHelper(context, activateContext);
 
+		await checkAuthLibraryOnStartup(context, activateContext, ext.loginHelper);
```

Comment:
I purposefully passed `ext.loginHelper` as a parameter here to enforce that class is instantiated before calling `checkAuthLibraryOnStartup`.